docs/topics/rn-resolved-issues.adoc

```diff
@@ -17,19 +17,23 @@ At the time of the release, the following resolved issues have been identified a
 |Component
 |Summary
 
-|link:https://issues.redhat.com/browse/WINDUP-3201[WINDUP-3201]
-|Red Hat CodeReady Studio extension
-|The plugin cannot be used in Red Hat CodeReady Studio 12.21.0.GA.
+|link:https://issues.redhat.com/browse/WINDUP-2622[WINDUP-2622]
+|Web console on OpenShift Container Platform
+|Custom rules do not work on OpenShift Container Platform.
 
-|link:https://issues.redhat.com/browse/WINDUP-3125[WINDUP-3125]
-|ItelliJ IDEA plugin and Visual Studio Code extension
-|The *Apply QuickFix* menu option is not disabled after a QuickFix is applied.
+|link:https://issues.redhat.com/browse/WINDUP-3234[WINDUP-3234]
+|Rules engine
+|When an analysis using source mode is run on an application source directory that has the target subdirectory, the contents of the target subdirectory are added to the analysis scope. This leads to duplication or possible errors in the results.
 
```

Comment:
@RichardHoch 
Just for my understanding: Is it correct that a source directory has a target subdirectory? Why sub?